### PR TITLE
model card updated for finetuning of llama models 7b,13b,70b

### PR DESCRIPTION
**Pull request by Vishal-p117**
Model card updated for finetuning of llama models 7b,13b,70b
1 Added new sample dataset and notebook path for text generation.
2 Also added FT text generation task related to model 7b & 13b in spec.yaml files

Kindly review and approve this PR.

Reference tested Runs :

llama-13b run: [neat_whale_y3n1zjxmd2 - Azure AI | Machine Learning Studio](https://ml.azure.com/experiments/id/c9b21507-154a-4657-acdd-c2add1a421b6/runs/neat_whale_y3n1zjxmd2?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourceGroups/training_rg/providers/Microsoft.MachineLearningServices/workspaces/train-finetune-dev-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

llama-7b run: [eager_yuca_zkqpjqjx70 - Azure AI | Machine Learning Studio](https://ml.azure.com/experiments/id/c9b21507-154a-4657-acdd-c2add1a421b6/runs/eager_yuca_zkqpjqjx70?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourceGroups/training_rg/providers/Microsoft.MachineLearningServices/workspaces/train-finetune-dev-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

@novaturient95 @skasturi

### CODE REVIEW

**Comment by vikasagrawal-ms** [comment]
@HrishikeshGeedMS Do we need version update for description changes?

**Comment by novaturient95** [comment]
> @HrishikeshGeedMS Do we need version update for description changes?

no Vikas

File: assets/models/system/Llama-2-13b/description.md [hunk 1]
```diff
@@ -102,7 +102,8 @@ Please see the Responsible Use Guide available at [https://ai.meta.com/llama/res
 
 Task|Use case|Dataset|Python sample (Notebook)|CLI with YAML
 |--|--|--|--|--|
-Text Classification|Emotion Detection|<a href="https://huggingface.co/datasets/dair-ai/emotion" target="_blank">Emotion</a>|<a href="https://aka.ms/azureml-ft-sdk-emotion-detection" target="_blank">emotion-detection.ipynb</a>| coming soon
+Text Generation|Summarization|<a href="https://huggingface.co/datasets/samsum" target="_blank">Samsum</a>|<a href="https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/finetune/Llama-notebooks/text-generation/summarization_with_text_gen.ipynb" target="_blank">summarization_with_text_gen.ipynb</a>| coming soon
```
Comment (novaturient95) [inline assets/models/system/Llama-2-13b/description.md:105]: Please add successful runs in the description

Comment (Vishal-p117) [reply assets/models/system/Llama-2-13b/description.md:105]: Added please check

**Review by amangupta26 — APPROVED** [review]

**Pull request merged**